webapp/components/posts_view_container.jsx

```diff
@@ -60,17 +59,14 @@ export default class PostsViewContainer extends React.Component {
         ChannelStore.addLeaveListener(this.onChannelLeave);
         PostStore.addChangeListener(this.onPostsChange);
         PostStore.addPostsViewJumpListener(this.handlePostsViewJumpRequest);
-        UserStore.addChangeListener(this.onUserChange);
+        $('body').addClass('app__body');
```

Comment:
This should actually be removed from the posts_view_container. I moved it to the channel_view in https://github.com/mattermost/platform/pull/2763